### ISSUE TITLE
automatically prompt for a specific version if a server does not support the selected version

### ISSUE THREAD

**Issue by recluse4615** [minetest-world/mtlauncher#1]
tl;dr -

- some servers only support specific versions of MT
- the launcher currently _has_ this information, but does not act on it

it would probably be best to modify `$selectedVersion` when selecting a server in the `ServerListTable.svelte`, that way the rest of the application can "catch on" and prompt to install the specific maximum version needed

**Comment by recluse4615** [comment]
https://github.com/minetest-world/mtlauncher/blob/main/src/routes/%2Blayout.svelte#L104

`server` here is a full server object, which _should_ include 2 fields:
- `proto_min`
- `proto_max`

the "recommended version" can be figured out like so:
```
import { getVersions } from '$lib/api/versions';

// ...
let server = { ... };
let recommendedVersion = getVersions().filter(i => i.branch.protocolVersion === server.proto_max && !i.branch.isDev);
```

the best way to do this would probably be to `subscribe` to `selectedServer` like so:
```
selectedServer.subscribe(val => {
    if (val.hasOwnProperty('proto_max')) {
         let recommendedVersion = getVersions().filter(i => i.branch.protocolVersion === server.proto_max && !i.branch.isDev);
         $selectedVersion = recommendedVersion;
    }
});
```
might need some testing/extra reactive-ness tho

**Comment by recluse4615** [comment]
resolved in `v0.0.2`
